integrations/terraform/Makefile

```diff
@@ -155,3 +154,16 @@ destroy:
 .PHONY: lint
 lint:
 	golangci-lint run -c ../../.golangci.yml
+
+.PHONY: docs
+docs: gen-tfschema install bin/tfplugindocs
+	./gen/docs.sh $(VERSION)
+
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+bin/tfplugindocs: go.mod $(LOCALBIN)
+	mkdir -p bin
+	GOBIN=$(LOCALBIN) go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
```

Comment:
Shouldn't this use our fork?

Reply:
`go install` reads the go.mod which has the replace statement. We need to invoke the tool with the original module name, or rename everywhere in our fork.